lib/components/atoms/Anchor/Anchor.style.js

```diff
@@ -2,22 +2,14 @@ import { css } from 'styled-components';
 
 export default css`
   cursor: pointer;
-  text-decoration: none;
-  color: ${props => props.theme.linkColorPrimary};
+  ${props => props.theme.behaviour.LINK_DECORATION};
```

Comment:
Drilling down the props may result in a cumbersome experience. 
2 ways to minimise: either send the reqd values separately or club multiple props in one arrow function

Reply:
I know it kooks repetitive but this is just going to produce CSS. I find this style much leaner.